### PR TITLE
Added necessary codes for Continuation In

### PR DESCRIPTION
**Pull request by JazzarKarim**
*Issue #:* /bcgov/entity#19258

*Description of changes:*
- Added all the necessary codes/names/types for continuation in

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: src/modules/corp-type-module/corp-type-module.ts [hunk 1]
```diff
@@ -12,6 +12,7 @@ export enum CorpTypeCd {
   BC_COMPANY = 'BC',
   BENEFIT_COMPANY = 'BEN',
   CONTINUE_IN = 'C',
+  BEN_CONTINUE_IN = 'CBEN',
```
Comment (severinbeauvais) [inline src/modules/corp-type-module/corp-type-module.ts:15]: This should have a corresponding record in `CorpInfoArray` below.

Comment (JazzarKarim) [reply src/modules/corp-type-module/corp-type-module.ts:15]: Thanks for catching that Sev.

**Review by severinbeauvais — APPROVED** [review]

**Comment by severinbeauvais** [comment]
Weird: there's a failed unit test.

**Comment by JazzarKarim** [comment]
> Weird: there's a failed unit test.

Re-running the failed job fixed that 🙌

**Comment by severinbeauvais** [comment]
> > Weird: there's a failed unit test.
> 
> Re-running the failed job fixed that 🙌

Possibly a missing await or await-next-tick, then. We can fix it later.

**Pull request merged**